fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4

```diff
@@ -966,6 +966,11 @@ sampleMethod
     | INTEGER_VALUE ROWS                                            #sampleByRows
     ;
 
+tableSnapshot
+    : FOR VERSION AS OF version=number
```

Comment:
The range of `number` is relatively wide, and we can use `INTEGER_VALUE`

Reply:
Fixed.